### DISCUSSION TITLE
Photo Export and Clearing Previous Entries

### DISCUSSION THREAD

**Discussion by igmellon** [PhenoApps/Field-Book#1116] [General]
When I choose to export active traits, it exports a url referring to the photo in my android storage, but not the photo itself. I can go back and manually move them into breedbase but it is still a pain. Is there something I am doing wrong? 

Also, when we are phenotyping, we are interested in clearing the previous rating to prevent bias and see how the rating changes through the season. Is there any way to clear the previous ratings between phenotyping sessions or upon export?

**Comment by trife** [comment]
Images cannot be embedded in CSV files. The alternatives are to bundle the images with the export to produce a single zip file containing everything or use BrAPI to upload the images when data is exported via BrAPI. I don't know the current support for either of these approaches in BreedBase, but @lukasmueller can comment.
